cmd/root.go

```diff
@@ -74,6 +74,21 @@ func init() {
 	flags.Int("couchdb-port", 5984, "couchdbdb port")
 	checkNoErr(viper.BindPFlag("couchdb.port", flags.Lookup("couchdb-port")))
 
+	flags.String("mail-host", "", "mail smtp host")
```

Comment:
Why not `localhost` for the default? Is it because the TLS certificate won't match in that case?